drivers/modem/modem_cmd_handler.h

```diff
@@ -75,6 +75,8 @@ struct modem_cmd_handler_data {
 
 	int last_error;
 
+	char *eol;
```

Comment:
This seems to be always a const string, so perhaps adding `const char *eol`? here.